packages/terra-avatar/src/variants/Avatar.jsx

```diff
@@ -58,7 +59,7 @@ const defaultProps = {
 class Avatar extends React.Component {
   constructor(props) {
     super(props);
-
+    objectFitImages();
```

Comment:
To better encapsulate the polyfill's effects, you could provide an element here to update the individual Avatar instance: https://github.com/bfred-it/object-fit-images#objectfitimagesimages-options

You'd have to do this inside componentDidMount (and probably componentDidUpdate, if changing the avatar props requires execution again).

Reply:
ComponentDidMount works fine, as changing the avatar props does not require execution again.